src/dapp-tests/integration/tests.sh

```diff
@@ -49,10 +49,52 @@ dapp_testnet() {
 
   # dynamic fee transaction (EIP-1559)
   seth send "$A_ADDR" "on()" --gas 0xffff --password /dev/null --from "$ACC" --keystore "$TMPDIR"/8545/keystore --prio-fee 2gwei --gas-price 10gwei
+
+  # clean up
+  killall geth
 }
 
 dapp_testnet
 
+# checks that seth send works with both checksummed and unchecksummed addresses
+seth_send_address_formats() {
+  TMPDIR=$(mktemp -d)
+
+  dapp testnet --dir "$TMPDIR" &
+  # give it a few secs to start up
+  sleep 180
```

Comment:
FYI, you can make this like 3-10 seconds and it still works well enough. Should make CI faster